src/main/resources/application-c100-form.yaml

```diff
@@ -5,6 +5,16 @@ uk:
         caseTypeofApplication:
           C100:
             mandatoryFields:
+              - previous_or_ongoingProceeding
+              - existingCase_onEmergencyProtection_Care_or_supervisioNorder
+              - exemption_to_attend_MIAM
+              - familyMember_Intimation_on_No_MIAM
+              - attended_MIAM
+              - NoMIAM_domesticViolence
+              - NoMIAM_childProtectionConcerns
+              - NoMIAM_Urgency
+              - NoMIAM_PreviousAttendance
+              - NoMIAM_otherReasons
               - child1_firstName
```

Comment:
pls move to optional fileds

Reply:
Removed conditional mandatory fields to Optional field group